virtcontainers/utils.go

```diff
@@ -7,15 +7,22 @@ package virtcontainers
 
 import (
 	"crypto/rand"
+	"errors"
 	"fmt"
 	"os"
 	"os/exec"
+	"path/filepath"
 )
 
 const cpBinaryName = "cp"
 
 const fileMode0755 = os.FileMode(0755)
 
+// Effective maximum Unix domain socket length
+//
+// See unix(7).
+const maxSocketPathLen = 107
```

Comment:
I wish there was some go system level const for this eh?
Found a maybe relevant ticket https://github.com/golang/go/issues/6895 - and, looks like @jessfraz was around this area back in 2015 maybe - so maybe can give some insight/input.